### ISSUE TITLE
Date_report

### ISSUE THREAD

**Issue by sadiekelly** [globaldothealth/outbreak-schema#3]
*Minimum EpiCore variable = Date case was first reported

The attribute Date_report, label 'Date case was first reported' was suggested to add to the day0 schema in the G.h curator manual. This was defined in the suggestion as 'e.g. date first included in a situation report or other official communication'.

Does this meaning align with the EpiCore variable 'Date case was first reported'? Could this mean here date of first consultation with a healthcare provider? It would be good to clarify any differences in meaning of these dates to ensure the variables are labelled clearly and not aligned when coded.

**Comment by lauramerson** [comment]
I believe the day0 'Date case was first reported'  and the EpiCore variable 'Date case was first reported'?  are aligned. The definition of 'date first included in a situation report or other official communication' applies. I would hope that this is also aligned with 'Date of first case identification' though this is not always the case. These are different from interview date, which could be at a later point in the subject journey.

**Comment by sadiekelly** [comment]
@lauramerson the ICC doesn't seem to capture date case was first reported. There is date of enrolment (date of first data capture), date of first symptom, date of current hospital admission and earliest date of previous admission. I think these could all be different to first reported date - should first reported date be added to the ICC?

**Comment by lauramerson** [comment]
I agree that it is missing in the ICC. As the ICC currently targeets hospitalised patients only, it is not a variable we've needed. If we move to community studies we'll add it - but for now I think date of presentation to hospital is sufficient for the hospitalised data.

**Comment by sadiekelly** [comment]
Existing date_report label is clear for our data sources and aligns with other toolkit forms, as date of healthcare consultation is captured separately in the toolkit forms. 
No changes to the day0 schema required for this item.

**Comment by sadiekelly** [comment]
Existing G.h day0 schema has a variable for Date of first consultation, so if this is explicitly provided it can be captured separately to the date the case was reported

**Comment by sadiekelly** [comment]
Recommendation: No changes required to existing schema variables.
